x-pack/plugins/cases/server/client/cases/delete.ts

```diff
@@ -54,8 +56,24 @@ export async function deleteCases(ids: string[], clientArgs: CasesClientArgs): P
     user,
     userActionService,
     logger,
+    authorization: auth,
+    auditLogger,
   } = clientArgs;
   try {
+    const cases = await caseService.getCases({ soClient, caseIds: ids });
+    const owners = new Set<string>();
+
+    for (const theCase of cases.saved_objects) {
+      owners.add(theCase.attributes.owner);
+    }
+
+    try {
+      await auth.ensureAuthorized([...owners.values()], Operations.deleteCase);
```

Comment:
Could we use the `ensureAuthorized` helper function instead of calling `auth.ensureAuthorized` here?

Reply:
I accidentally pushed the delete route to this PR. That's why it is incomplete. I am gonna improve it.

Reply:
Done!